src/chttpd/src/chttpd.erl

```diff
@@ -448,8 +448,8 @@ fix_uri(Req, Props, Type) ->
         true ->
             Props;
         false ->
-            Uri = make_uri(Req, quote(Uri0)),
-            [{Type,Uri}|proplists:delete(Type,Props)]
+            Msg = "Local endpoints not supported since CouchDB 3.x",
```

Comment:
I'm on board with all of this other than removing the auto-convert to http endpoints and throwing errors.

Reply:
Fair enough. The `_replicate` actually works currently, albeit it is inconsistent with what `_replicator` docs do. Let's keep it for 3.x, mark as deprecated and remove it for 4.x